docs/manual/examples/quick-batching.qdoc

```diff
@@ -62,40 +62,42 @@
-    transformation.
+    Comparing the subtrees belonging to active and inactive rectangles scenegraph-node tree view,
+    we notice that the inactive ones have an Opacity node, while the active ones don't. The opacity
+    node indicates transparency.
 
-    In order to match this to the actual code causing the problem, switch back to the item tree view,
-    and use the context menu action "Go to declaration", which will open a code editor around the
-    following piece of code:
+    now we can check the opacity property in the \l {Properties} tab of the Item view, so we switch back to
+    "Items". The Item corresponding to the previously selected Opacity node is already pre-selected, so
+    we only need to look for the property. In fact we find it to have value < 1.
 
-    \snippet quick-batching/Slider.qml Slider transformation
+    In order to match this to the actual code causing the problem, use the context menu action
+    "Go to declaration", which will open a code editor around the following piece of code:
 
-    In order to verify that this is our problem you don't need to restart the application, you can also
-    use the editing capabilities of the \l{Properties} tab in the item tree view to adjust the rotation live.
-    After doing this we see that the left slider is no having a trivial diagonal in its combined
-    transformation matrix, and thus is rendered with a single OpenGL draw call.
-    For the right slider we have to repeat this procedure to identify a second transformation:
+    \snippet quick-batching/Slider.qml Slider delegate
 
-    \snippet quick-batching/Slider.qml Slider mirroring
+    In fact we see that the opacity value is set to a value depending on the index. That means that all
+    rectangles get a different opacity value and that rules out batching.
+    In order to verify that this is actually our problem you don't need to restart the application,
+    you can also use the editing capabilities of the \l{Properties} tab in the item tree view to adjust
+    the opacity live. If you still have the batch rendering visualization activated, after setting the
+    opacity to 1, you'll observe how the selected rectangle gets the same color as all the active
+    rectangles.
 
-    Just removing the transformations is of course not a proper fix (as it changes the visual appearance
-    of the application), it merely verifies that it's these rotations causing our performance problem.
-    You would now need to refactor the code in a way that it achieves the same behavior and visual
-    appearance without the use of rotational transformations.
+    Just removing the opacity is of course not a proper fix, it merely verifies that it's these opacities
+    causing our performance problem.
+    To retain the same visual appearance as with opacity set, you could e.g. use the alpha component of
+    the color property to obtain the transparency, without impairing the batching.
```

Comment:
Does this actually work, in the given example? Does it exploit the fact that the items don't overlap, so they can be batched in the alpha pass?

Reply:
Yes it did work (previously I wrote, it wouldn't until I realized it does actually) but no, it doesn't exploit the fact that the items don't overlap. I explicitly made them overlap with each rectangle having a unique alpha-value, still it was able to batch them (or at least the batch-visualizer showed them as batched)